Cargo.toml

```diff
@@ -57,7 +57,7 @@ path = "examples/hello_world/server.rs"
 serde_json = "1.0"
 serde = "1.0"
 serde_derive = "1.0"
-grpcio-proto = { path = "proto", version = "0.5.0-alpha" }
+grpcio-proto = { path = "proto", features = ["prost-codec"], default-features = false }
```

Comment:
Why use prost-codec features?

Reply:
It was easiest (because this what was done already)? I think it could support both, but I think for an example it is maybe OK to pick one.

Reply:
Then how can we know our default codecs work as expected if it's not being used by any crates?

Reply:
Fair. We can probably use both codecs

Reply:
I think we can just use rust-protobuf for now as first step.